docs/en/operations/storing-data.md

```diff
@@ -464,9 +466,11 @@ In this sample configuration:
 </clickhouse>
 ```
 
+Keep in mind that HDFS does not work at all.
```

Comment:
May be this message is too much? "At all" is not really true... If if did not work at all, there would have been no sense having hdfs in documentation at all.

Reply:
@kssenii latest HDFS have a hard limit 1million files per folder. 
Clickhouse stores all objects in the the single HDFS folder (for HDFS as external storage, unlike S3).
The average table with 500 columns reaches this limit 1million objects easily, it's less 1000 active+inactive wide parts, after that a table is unable to insert or to merge.

https://github.com/ClickHouse/ClickHouse/issues/43112